Formula/vectorscan.rb

```diff
@@ -0,0 +1,40 @@
+class Vectorscan < Formula
+  desc "High-performance regular expression matching library"
+  homepage "https://github.com/VectorCamp/vectorscan"
+  url "https://github.com/VectorCamp/vectorscan/archive/refs/tags/vectorscan/5.4.7.tar.gz"
+  sha256 "cd70c2a7bf632b5374083a450019703605520c10c5614a4c12011c99ab8435dd"
+  license "BSD-3-Clause"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ragel" => :build
+  depends_on arch: :arm64
+  depends_on "pcre"
+
+  def install
+    cmake_args = std_cmake_args + [
+      "-DBUILD_STATIC_AND_SHARED=ON",
+      "-DPYTHON_EXECUTABLE:FILEPATH=/usr/bin/python3",
```

Comment:
Do we need to hard-code this?

Reply:
There needs to be _some_ change since /usr/bin/python was removed from macos Monterey at some point > 12.2.1 and <= 12.4.  As far as I can tell, the alternative is to update the cmake configuration files.

Reply:
How picky is it about the version of Python 3? Could we do just `python3` (without the `/usr/bin` prefix) or perhaps `/usr/bin/env python3`?

Reply:
`python3` on its own works fine.